org.libvips.vipsdisp.json

```
@@ -0,0 +1,341 @@
+{
+    "app-id" : "org.libvips.vipsdisp",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "vipsdisp",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfsd",
+        "--filesystem=home"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/girepository-1.0",
+        "/lib/pkgconfig",
+        "/lib/cmake",
+        "/lib/debug",
+        "/lib/openjpeg-*",
+        "/lib/libhdf5.settings",
+        "/man",
+        "/share/aclocal",
+        "/share/doc",
+        "/share/gir-1.0",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/aclocal*",
+        "/share/autoconf*",
+        "/share/automake*",
+        "/share/info",
+        "/share/thumbnailers",
+        "/share/USING_HDF5_CMake.txt",
+        "/share/COPYING",
+        "/share/RELEASE.txt",
+        "/share/cmake",
+        "/bin/a*",
+        "/bin/b*",
+        "/bin/c*",
+        "/bin/d*",
+        "/bin/e*",
+        "/bin/f*",
+        "/bin/g*",
+        "/bin/h*",
+        "/bin/i*",
+        "/bin/l*",
+        "/bin/m*",
+        "/bin/o*",
+        "/bin/p*",
+        "/bin/r*",
+        "/bin/s*",
+        "/bin/t*",
+        "/bin/x*",
+        "/bin/vips",
+        "/bin/vipsheader",
+        "/bin/vipsedit",
+        "/bin/vips-*",
+        "/bin/vipsprofile",
+        "/bin/vipsthumbnail",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+
+        {
+            "name" : "libjxl",
+            "builddir" : true,
+            "buildsystem" : "cmake-ninja",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/libjxl/libjxl.git",
+                    "tag" : "v0.5",
+                    "commit" : "c4e0877f93506e880cd922f6c94644d79ae9adff"
+                }
+            ]
+        },
+
+        {
+            "name" : "openjpeg",
+            "builddir" : true,
+            "buildsystem" : "cmake-ninja",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/uclouvain/openjpeg.git",
+                    "tag" : "v2.4.0",
+                    "commit" : "37ac30ceff6640bbab502388c5e0fa0bff23f505"
+                }
+            ]
+        },
+
+        {
+            "name" : "libimagequant",
+            "builddir" : true,
+            "buildsystem" : "cmake-ninja",
```

```suggestion
            "buildsystem" : "cmake-ninja",
            "build-options": {
                "arch": {
                    "aarch64": {
                        "config-opts": [
                            "-DBUILD_WITH_SSE=NO"
                        ]
                    }
                }
            }
```

tl;dr, another library that assume they build for an x86_64 platform.

---

this is untested.

---

That's great! I'll try it.